.env.example

```
@@ -55,3 +55,30 @@ SMTP_FROM=EveryCal <noreply@yourdomain.com>
 # Legacy dev toggle (not required for non-production link logging)
 # SKIP_EMAIL_VERIFICATION=true
+
+# OIDC SSO (optional; disabled by default). Generic OpenID Connect providers such as Authentik work with these fields.
+OIDC_ENABLED=false
+OIDC_PROVIDER_KEY=primary_oidc
+# OIDC_ISSUER_URL=https://idp.example.com/application/o/everycal/
+# OIDC_CLIENT_ID=everycal
+# OIDC_CLIENT_SECRET=replace-with-provider-secret
+# Defaults to ${BASE_URL}/api/v1/auth/oidc/callback when omitted.
+# OIDC_REDIRECT_URI=https://events.example.com/api/v1/auth/oidc/callback
+OIDC_SCOPES=openid profile email
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)